code/modules/cargo/packs/medical.dm

```diff
@@ -48,6 +48,13 @@
 	small_item = TRUE
 	contains = list(/obj/item/storage/firstaid/radiation)
 
+/datum/supply_pack/medical/ifak
```

Comment:
This seems to be an unrelated change to the ruin.

Reply:
tangentially related item from adhd-coding. can cut